src/Components/CustomButton/CustomButton.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { css, StyleDeclaration, StyleSheet } from "aphrodite";
+
+export interface CustomButtonProps {
+  onClick: () => void;
+  label: string;
+  customStyles: StyleDeclaration;
+  disabled: boolean;
+  filled: boolean;
```

Comment:
They are optional props, so if you do not pass them, they will be false :)

-> customStyles?: ....   disabled?: boolean   filled?: boolean